README.md

```diff
@@ -15,7 +15,9 @@ ManageIQ Python API Client package [manageiq-api-client-python] (https://github.
 
 ### Getting Started
 
-Currently, the module supports adding an OpenShift containers provider to manageiq.
+###### manageiq_provider module
+
+The `manageiq_provider.py` module currently supports adding, updating and deleting an OpenShift containers provider to manageiq.
```

Comment:
@kwoodson - we usually add it when formally describing OpenShift, but I don't mind removing it, thanks.
@cben - I agree, thanks